api/backup.go

```diff
@@ -54,6 +109,61 @@ func (s *Server) BackupVolumeGet(w http.ResponseWriter, req *http.Request) error
 	return nil
 }
 
+func (s *Server) BackupVolumeUpdateAll(w http.ResponseWriter, req *http.Request) error {
+	var input BackupVolume
+
+	apiContext := api.GetApiContext(req)
+	if err := apiContext.Read(&input); err != nil {
+		return err
+	}
+
+	bvs, err := s.m.ListBackupVolumesSorted()
+	if err != nil {
+		return errors.Wrap(err, "failed to list backup volumes")
+	}
+
+	if input.SyncBackupVolume {
+		for _, bv := range bvs {
+			if _, err := s.m.SyncBackupVolume(bv); err != nil {
+				return errors.Wrapf(err, "failed to update backup volume '%s'", bv.Name)
```

Comment:
Shall we error out here or just do best effort?

Reply:
We could have a warning log if there is an error with synchronizing the backup volume and continue synchronizing the next backup volume.
WDYT?

Reply:
Modified to leave a warning log instead of returning an error and modified the part of backup target as well. Thanks